packages/compiler/src/core/cli/cli.ts

```diff
@@ -208,6 +208,10 @@ async function main() {
           .option("template", {
             type: "string",
             description: "Name of the template to use",
+          })
+          .option("skip-install", {
```

Comment:
A more common name would be `no-install` i think

e.g. astro template init
```
--install / --no-install  Install dependencies (or not).
```

Reply:
Updated. The yargs flag is `--install` which automatically enables `--no-install`